pjmedia/include/pjmedia/sound_port.h

```diff
@@ -71,7 +71,15 @@ enum pjmedia_snd_port_option
     /** 
      * Don't start the audio device when creating a sound port.
      */    
-    PJMEDIA_SND_PORT_NO_AUTO_START = 1
+    PJMEDIA_SND_PORT_NO_AUTO_START = 1,
+
+    /**
+     * Sound device uses \ref PJMEDIA_CLOCK instead of native sound device
+     * clock, generally this will be able to reduce jitter and clock skew.
+     *
+     * This option is not applicable for encoded/non-PCM format.
+     */
+    PJMEDIA_SND_PORT_USE_SOFT_CLOCK = 2
```

Comment:
Perhaps using SW_CLOCK would be more intuitive? It's inline with what we've been using, such as USE_SW_ECHO, SW_VID_ENC/DEC.